ONSdigital/dp-healthcheck
# Downgrade to version 1, leaving the old signature for AddCheck, and c…

### What

Downgrade dp-healthcheck to version 1, fixing the broken signature change that was the reason why we had upgraded it to v2:
- Leave the old signature for `AddCheck`
- Create a new Method AddAndGetCheck, for the new functionality
- Remove `v2` from package name in go.mod, and imports in the code.

### How to review

- Make sure code changes make sense
- Make sure the `AddCheck` signature matches old v1 `AddCheck`
- Make sure unit tests pass

### Who can review

Anyone
